alicloud/resource_alicloud_yundun_bastionhost_instance_test.go

```diff
@@ -0,0 +1,291 @@
+package alicloud
+
+import (
+	"fmt"
+	"log"
+	"strings"
+	"testing"
+
+	"github.com/aliyun/alibaba-cloud-sdk-go/sdk/requests"
+	"github.com/aliyun/alibaba-cloud-sdk-go/services/yundun_bastionhost"
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/terraform-providers/terraform-provider-alicloud/alicloud/connectivity"
+)
+
+func init() {
+	resource.AddTestSweepers("alicloud_yundun_bastionhost_instance", &resource.Sweeper{
+		Name: "alicloud_yundun_bastionhost_instance",
+		F:    testSweepBastionhostInstances,
```

Comment:
The bastionhost instance depends on vswitch and secuity group. So the vswitch and sg sweeper test should add bashion host sweeper. Refer to https://github.com/terraform-providers/terraform-provider-alicloud/blob/master/alicloud/resource_alicloud_vswitch_test.go#L25